web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -107,3 +108,89 @@ function dpl_webform_update_10004(): string {
+  $feedback = [];
+
+  \Drupal::configFactory()->getEditable('system.site')->set('langcode', 'en')->save();
+  \Drupal::configFactory()->getEditable('system.site')->set('default_langcode', 'en')->save();
+
+  return implode("\n", $feedback);
+}
+
+/**
+ * Remove unwanted webform settings from config ignore.
+ *
+ * Some libraries has been able to add unwanted webform settings to the
+ * config ignore list. This is preventing us from fixing some faulty
+ * webform translation configurations. This update hook will remove all
+ * webform.webform_options.* and webform.settings from the ignore list.
+ */
+function dpl_webform_update_10006(): string {
+  $config_factory = \Drupal::configFactory();
+  $config_ignore_auto_settings = $config_factory->getEditable('config_ignore_auto.settings');
+  $ignored_configs = $config_ignore_auto_settings->get('ignored_config_entities');
+
+  // Remove webform.settings from config ignore list.
+  if (($key = array_search('webform.settings', $ignored_configs)) !== FALSE) {
+    unset($ignored_configs[$key]);
+    $config_ignore_auto_settings->set('ignored_config_entities', $ignored_configs)->save();
+    $feedback[] = "Removed 'webform.settings' from ignored_config_entities.";
+  }
+  else {
+    $feedback[] = "Config 'webform.settings' not found in ignored_config_entities.";
+  }
+
+  // Filter out all strings matching the pattern webform.webform_options.*.
+  $pattern_to_remove = '/^webform\.webform_options\..*/';
+  $filtered_configs = array_filter($ignored_configs, function ($config) use ($pattern_to_remove) {
+    return !preg_match($pattern_to_remove, $config);
+  });
+
+  if (count($ignored_configs) !== count($filtered_configs)) {
+    $config_ignore_auto_settings->set('ignored_config_entities', array_values($filtered_configs))->save();
+    $feedback[] = "Removed all strings matching 'webform.webform_options.*' from ignored_config_entities.";
+  }
+  else {
+    $feedback[] = "No strings matching 'webform.webform_options.*' found in ignored_config_entities.";
+  }
+  return implode("\n", $feedback);
+}
+
+/**
+ * Rerun old update hooks.
+ *
+ * Some earlier update hooks did not have the desired effect due to
+ * some configuration ignore issues. We will rerun the update hooks
+ * here again, to make sure that we have deleted all faulty translations.
+ * We rerun update_10002 to delete existing contact webform translations.
+ * We rerun update 10004 to make sure to delete faulty translations with
+ * duplicate keys.
+ */
+function dpl_webform_update_10007(): string {
+
+  $feedback = [];
+
+  try {
+    $feedback[] = dpl_webform_update_10002();
+  }
+  catch (\Throwable $t) {
+    $feedback[] = "Unable to rerun dpl_webform_update_10002: {$t->getMessage()}.";
+  }
+
+  try {
+    $feedback[] = dpl_webform_update_10004();
+  }
+  catch (\Throwable $t) {
+    $feedback[] = "Unable to rerun dpl_webform_update_10004: {$t->getMessage()}.";
+  }
+
+  return implode("\n", $feedback);
+}
```

Comment:
I do not agree in this approach.
I think we should always move forward. If you need to redo previous actions simply make new update hooks repeating the old action. Update hooks are imperative by nature - I think.

Reply:
I guess that makes sense and my first approach was also to do that. But when I took a look at how we have done similar things earlier, I decided to copy the same approach.

However I have now updated the update hook, to include the previously run code from earlier update hooks instead of calling the update hook functions.